packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -72,7 +72,6 @@ library ChainAssertions {
         ResourceMetering.ResourceConfig memory resourceConfig = config.resourceConfig();
 
         if (_isProxy) {
-            require(config.owner() == _cfg.finalSystemOwner());
             require(config.basefeeScalar() == _cfg.basefeeScalar());
```

Comment:
Rather then deleting this check - can we update it so it checks the owner based on config ?